src/app/components/wrapper-components/local-game-wrapper/local-game-wrapper.component.ts

```diff
@@ -44,7 +56,17 @@ export class LocalGameWrapperComponent extends GameWrapper<string> implements Af
         super(actRoute, connectedUserService, router, messageDisplayer);
         this.players = [MGPOptional.of(this.playerSelection[0]), MGPOptional.of(this.playerSelection[1])];
         this.role = Player.ZERO; // The user is playing, not observing
+        this.setDefaultRulesConfig();
     }
+
+    // Will set the default rules config
+    // Will set it to {} if the game don't exist, but an error will be handled by some other function
```

Comment:
wat, that's not what it does?

Reply:
yes it is

Reply:
à quelle ligne tu check que le jeu n'existe pas et mets la config à `{}` ?!

Reply:
RulesConfigUtils.getGameDefaultConfig(gameName); le fera

Reply:
... si le jeu existe pas, son champ rules peut pas exister ?! (cf. ligne suivante "    // Because rules that don't override their rules have DEFAULT_RULES value, hence {}")

BTW: if the game **doesn't** exist

Reply:
````
public static getGameDefaultConfig<C extends RulesConfig>(gameName: string): C {
        const gameInfos: GameInfo[] = GameInfo.getByUrlName(gameName);
        let rulesConfigDescription: RulesConfigDescription = RulesConfigDescription.DEFAULT;
        if (gameInfos.length > 0) {
            rulesConfigDescription = gameInfos[0].getRulesConfigDescription();
        }
        return rulesConfigDescription.getDefaultConfig().config as C;
    }
````
Le behavior souhaité est là et c'est du code propre, donc cette fonction retourne {} quand le jeux existe pas ça simplifie tout